packages/ui/src/pages/Home.tsx

```diff
@@ -394,6 +415,8 @@ export default styled(Home)(
 
   .pureHeader {
     margin-bottom: 1rem;
+    display: flex;
+    justify-content: space-between;
```

Comment:
We need to add a `margin-left: 0.5rem` for mobile on this one, so that it's not stuck to the left
![image](https://github.com/ChainSafe/Multix/assets/33178835/ddb36090-385c-4a74-be27-536ce216dd1d)

Reply:
fixed